chore(inputs): adjust icons

mukunku opened the pull request:
# Summary

This PR adjusts the alert icons for Inputs and Selects with an error state. It also updates all icons with new versions available in the Svelte component to the latest beta revision.

# How to Test

## Main Changes

The main purpose of this PR was to update the Alert icons for Input, TextArea, and Select:

* Check out the new error state icon for `input`: https://deploy-preview-2103--stacks.netlify.app/product/components/inputs/#error
* Check out the new error state icon for `select`: https://deploy-preview-2103--stacks.netlify.app/product/components/select/#error

## Bonus Changes

I additionally updated other icons to their newer versions where available:

* Check buttons with the new trash icon: https://deploy-preview-2103--stacks-svelte.netlify.app/?path=/story/components-button--additional-styles
* Check notice close and alert icons: https://deploy-preview-2103--stacks-svelte.netlify.app/?path=/docs/components-notice--docs#actions
* Check post summary ellipsis icon: https://deploy-preview-2103--stacks-svelte.netlify.app/?path=/docs/components-postsummary--docs#optional-slots
* Confirm tag dismiss button looks good: https://deploy-preview-2103--stacks-svelte.netlify.app/?path=/docs/components-tag--docs#additional-styles-1
* Confirm toast error icon looks good: https://deploy-preview-2103--stacks-svelte.netlify.app/?path=/docs/components-toast--docs#with-icon-3
* Check user card deleted user avatar icon: https://deploy-preview-2103--stacks-svelte.netlify.app/?path=/story/components-usercard--deleted

mukunku committed:
adjust alert icons for inputs and select

mukunku committed:
Create seven-dryers-smile.md

CGuindon reviewed (changes requested):
LGTM on Classic — the Svelte side would need to be updated as well and match what you did.

<img width="1256" height="515" alt="Screenshot 2025-12-16 at 12 57 41 PM" src="https://github.com/user-attachments/assets/74bae676-ae94-44f2-96e5-c44caea9390e" />

mukunku committed:
update all icons we can in Svelte

mukunku committed:
lint

mukunku committed:
fix tests

mukunku committed:
fix tests

mukunku commented:
Thanks for the review @CGuindon . I updated the Svelte components now as well. I updated, not just the alert icons but, all the icons I could find that use the legacy icons that have a new version available. Let me know what you think 🙏🏾

mukunku committed:
update stacks icons and use IconMoreV for ellipses

mukunku committed:
lint
